### PR TITLE
Cube browser recipe.

### PR DESCRIPTION
**Pull request by bjlittle**

### CODE REVIEW

File: cube_browser/bld.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+python setup.py install --single-version-externally-managed --record record.txt
```
Comment (pelson) [inline cube_browser/bld.bat]: This magic line requires setuptools (not a core python library). Either cube_browser depends on setuptools, or this line can simply be `python setup.py install`.

File: cube_browser/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+if [[ $(uname) == "Darwin" ]]; then
```
Comment (pelson) [inline cube_browser/build.sh]: This if statement isn't used, so the Windows and Unix build commands are the same. In which case, we can simply make this part of the `meta.yaml` under the `build/script` section. http://conda.pydata.org/docs/building/meta-yaml.html#script

File: cube_browser/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - python
+    run:
+        - iris 1.10*
+        - ipywidgets 4.1.1
```
Comment (pelson) [inline cube_browser/meta.yaml]: Is this really `4*`? or even `4.1*`?

File: cube_browser/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        - matplotlib
+
+test:
+    imports:
+        - cube_browser
+    
+about:
+    home: http://scitools.org.uk/cube_browser
+    license: BSD
```
Comment (pelson) [inline cube_browser/meta.yaml]: FWIW, we specify that it is `BSD 3-clause` in conda-forge.

**Comment by pelson** [comment]
Looks like the CI isn't picking up the `dev` channel. I guess for the dev branch, we need to add it as per https://github.com/SciTools/conda-recipes-scitools/blob/dev/.travis.yml#L37.

**Comment by pelson** [comment]
(and all other CI services will have a similar approach)

File: scripts/run_docker_build.sh [hunk 5]
```diff
@@ -44,6 +44,8 @@ conda install --yes conda-build=1.18.2
 conda info
 unset LANG
 yum install -y expat-devel git autoconf libtool texinfo check-devel gcc-gfortran
+conda config --add channels http://conda.binstar.org/SciTools
```
Comment (pelson) [inline scripts/run_docker_build.sh]: Curious. How did this work before?

Comment (pelson) [inline scripts/run_docker_build.sh]: https://github.com/SciTools/conda-recipes-scitools/blob/master/scripts/run_docker_build.sh#L9-L18 :wink:

**Comment by bjlittle** [comment]
@pelson :smile:

**Comment by pelson** [comment]
So it looks like we don't have a cartopy for py35 on Windows:

```
conda.resolve.Unsatisfiable: The following specifications were found to be in conflict:
  - cartopy -> numpy 1.8*
  - cartopy -> python 2.7*|3.4*
  - python 3.5*
Use "conda info <package>" to see the dependencies for each package. 
```

Not your problem, and it has been solved by conda-forge, so let's go with what we have already.

**Pull request merged**